dashboard/consts.py

```diff
@@ -52,3 +52,8 @@
 # ray.dashboard.modules.snapshot.snapshot_head.RayActivityResponse.
 # Example: "your.module.ray_cluster_activity_hook".
 RAY_CLUSTER_ACTIVITY_HOOK = "RAY_CLUSTER_ACTIVITY_HOOK"
+
+# The number of candidate agents
+CANDIDATE_AGENT_NUMBER = max(env_integer("CANDIDATE_AGENT_NUMBER", 1), 1)
+WAIT_RAYLET_START_TIMEOUT_SECONDS = 10
+WAIT_RAYLET_START_INTERVAL_SECONDS = 0.1
```

Comment:
Could you add more comments about how this constants are used?
Basically, we need to make sure people will understand the constants by just reading the name or comments.

Reply:
> @Catch-Bull could you give the PR a better name and description? It's really confusing about what this one is doing just from the name.

done.